commands/createfixpullrequests.go

```diff
@@ -164,36 +164,26 @@ func (cfp *CreateFixPullRequestsCmd) fixIssuesSeparatePRs(fixVersionsMap map[str
 	return
 }
 
+// Fix all the issues in one pull request.
+// In case of existing aggregated fix, check for different scan results
+// if scan results are the same, do nothing.
+// Else, force push to the same branch to update
+// Only one aggregated pull request should be open at all times.
 func (cfp *CreateFixPullRequestsCmd) fixIssuesSinglePR(fixVersionsMap map[string]*utils.FixDetails) (err error) {
```

Comment:
We should design a test that simulate and verifies all of the possible scenarios -
1. Identical results to an existing PR
2. Different results from an existing PR
3. No existing open PR

Reply:
I'll implement those tests in https://github.com/jfrog/frogbot/pull/345
As all of the testings infrastructure exists there.